data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+			choice
+				`	"Yes"`
+					goto ContinueSalv
+				`	"Sorry, not interested."`
+					decline
+			label "ContinueSalv"
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumours that someone was building a giant library out in the Deep. If it still exists, that would likely be where to go."`
+				accept
+	on stopover
+		conversation
+			`After having traveled clear across the galaxy, you finally have arrived at the archives of human history. On the direction of the librarians you locate the gift shop and buy a "souvenir copy" of the archive. You idly wonder if you should buy a couple spare copies, just in case, but decide against it.` 
+	
+	on complete
+		conversation
+			`Having literally crossed the galaxy for a copy of the archive, you gently set <ship> down on Viminal hoping that this information is helps. The port seems unusually busy, but the comm channels are quiet beyond the bare minimum to keep traffic organized.`
+			`	Before you have even taken the time to shutdown the engines Tali is halfway up the side of your ship securing a refueling hose and charging cable. As you reach for the comlink she scrambles up to the bridge windows and quickly rattles off a series of signs at you. All you can make out is "Keep ... hot, incoming something?" Before you can frame a reply she has raced across your wing and leapt to the Albatross parked next to you.`
+			`	Looking around more carefully, you note that every ship in the port is idling with fuel lines still connected. If this was some Republic port the safety inspectors would be having a nightmares. One Starling is getting holes in its wings patched even as the crew appear to be running prep-for-flight checklists. It also looks like every ship has a guard. Something tells you that right now would be a bad time to take a walk around town.`
+
+mission "Remnant: Salvage 2"
+	landing
+	source "Viminal"
+	name "Defend <planet>"
+	description "Assist in defending <planet> from a Korath raid, then return to the planet."
+	to offer
+		has "Remnant: Salvage 1: done"
+	on offer
+		conversation
+			`In the flurry of activity and tense waiting for something, it is almost a relief when the planetary defence sirens start. As you watch, the method to their chaos is suddenly evident: Each guard or port hand who had been standing around reaches up and pulls the quick release cable attached to the fuel hose, then race to get themselves and the hoses back out of the way. Mechanics operating overhead slap the buttons on their harnesses to be whisked up to the ceiling of the dome at breakneck speeds. Within seconds the first ships are out of the dome and the rest follow closely behind.`
```

Comment:
"defence" to "defense" (#3476)

Reply:
Done. For some reason, while gray vs grey doesn't matter to me, I get annoyed reading "defense." Amusing, it is.